src/ir/properties.h

```diff
@@ -26,6 +26,24 @@ namespace wasm {
 
 namespace Properties {
 
+inline int getBits(const Type& type) {
```

Comment:
Looks like https://github.com/WebAssembly/binaryen/blob/1e79c53ec1ad3d80db3354f15919331fdfa7ed28/src/wasm-type.h#L193-L194 ? (but in bits and not bytes)

Reply:
Oh right! Totally forget about `getByteSize`!